shaas.go

```diff
@@ -88,6 +90,28 @@ func handleExit(res http.ResponseWriter, req *http.Request) {
 	os.Exit(code)
 }
 
+func handleHealth(res http.ResponseWriter, req *http.Request) {
+	responseCode := 200
+	var err error
+	if c := req.URL.Query().Get("responseCode"); c != "" {
```

Comment:
what would you think making this work for any endpoint? (not just `/health`)

Reply:
for some nitpicky reason I can't explain, the camel case `responseCode` bothers me :) what about a shorter `code` or `status`? (or just feel free to ignore my irrationally ;))

Reply:
yeah I don't particularly care one way or the other, I didn't want to use `code` because the exit endpoint uses `code` but for a different purpose, I can go with `status`